### ISSUE TITLE
Blob key = null on AppEngine

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#119]
```
What steps will reproduce the problem?
I just added: 
addOnFinishUploadHandler(new IUploader.OnFinishUploaderHandler() {
}

And within that, uploader.getServerInfo().message is null.

On local server it works perfectly.

What is the expected output? What do you see instead?
I would like to see an blob key... I see null.

What version of the product are you using? On what operating system?
Latest, freshly checked-out from source repository.


Please provide any additional information below.


```

Original issue reported on code.google.com by `m.bartc...@gmail.com` on 26 Aug 2011 at 2:19

**Comment by GoogleCodeExporter** [comment]
```
%%%START%%%@@@?xml version="1.0" encoding="UTF-8"?___
@@@response___@@@field___GWTU-05623378876266829@@@/field___
@@@name___A box and a button.ppt@@@/name___
@@@ctype___application/vnd.ms-powerpoint@@@/ctype___
@@@finished___ok@@@/finished___
@@@size___1454080@@@/size___
@@@/response___
%%%END%%%

This is what I get from real appengine on upload, while on dev session I get:

%%%START%%%@@@?xml version="1.0" encoding="UTF-8"?___
@@@response___@@@message___
@@@![CDATA[
w5XIFSP_WRr66fdZzG8ArA
]]___
@@@/message___
@@@field___GWTU-06315443779113006@@@/field___
@@@name___A box and a button.ppt@@@/name___
@@@ctype___application/vnd.ms-powerpoint@@@/ctype___
@@@finished___ok@@@/finished___
@@@size___1454080@@@/size___
@@@/response___
%%%END%%%

Simply put - the content is gone.
```

Original comment by `m.bartc...@gmail.com` on 26 Aug 2011 at 2:28

**Comment by GoogleCodeExporter** [comment]
```
And to make it even easier - it works on Chrome, fails on Firefox, so it might 
be some kind of browser-related issue.
```

Original comment by `m.bartc...@gmail.com` on 26 Aug 2011 at 2:33

**Comment by GoogleCodeExporter** [comment]
```
Could you check with last snapshot?

thanks
- Manolo
```

Original comment by `manuel.carrasco.m` on 1 Jun 2012 at 10:30

**Comment by GoogleCodeExporter** [comment]
```
It works with either 0.6.5 or 0.6.6-SNAPSHOT
```

Original comment by `manuel.carrasco.m` on 3 Dec 2012 at 12:25
- Changed state: **Verified**
